integration-test-common/build.gradle.kts

```diff
@@ -43,6 +43,8 @@ dependencies {
   testImplementation(libs.httpclient5)
   testImplementation(libs.testcontainers)
   testImplementation(libs.testcontainers.mysql)
+  testImplementation(libs.mysql.driver)
+  testImplementation(libs.postgresql.driver)
```

Comment:
why add these dependencies?

Reply:
AbstractIT in `integration-test-common` needs `MySQL` and `PG` to start the Gravitino server. If we want to use it in some test like `SparkQueryRunner`, the dependency is missing.

Reply:
Why does it work before then?

Reply:
I removed these two dependencies.